data-source/blocks/individual/identity-and-health/language.jsonnet

```diff
@@ -1,110 +1,86 @@
 local placeholders = import '../../../lib/placeholders.libsonnet';
 local rules = import '../../../lib/rules.libsonnet';
 
-local question(proxyOptions, regionOptions) = {
+local question(title, definitionContent, regionOptions) = {
   id: 'language-question',
-  title: proxyOptions.title,
+  title: title,
   type: 'General',
-  definitions: proxyOptions.definitionContent,
+  definitions: [definitionContent],
   answers: [
     {
       id: 'language-answer',
       mandatory: true,
       type: 'Radio',
-    } + regionOptions,
-  ],
-};
-
-local nonProxyOptions = {
-  title: 'What is your main language?',
-  definitionContent: [
-    {
-      title: 'What do we mean by “main language”?',
-      content: [
+      options: [
+        regionOptions,
         {
-          description: 'Your main language is the language you use most naturally. It could be the language you use at home.',
+          label: 'Other',
+          value: 'Other',
+          description: 'Including British Sign Language',
+          detail_answer: {
+            id: 'language-answer-other',
+            type: 'TextField',
+            mandatory: false,
+            label: 'Please specify main language',
+          },
         },
       ],
     },
   ],
 };
-local isProxyOptions = {
-  title: {
-    text: 'What is <em>{person_name_possessive}</em> main language?',
-    placeholders: [
-      placeholders.personNamePossessive,
-    ],
-  },
-  definitionContent: [
+
+local nonProxyTitle = 'What is your main language?';
+local nonProxyDefinition = {
+  title: 'What do we mean by “main language”?',
+  content: [
     {
-      title: 'What do we mean by “main language”?',
-      content: [
-        {
-          description: 'Your main language is the language they use most naturally. It could be the language they use at home.',
-        },
-      ],
+      description: 'Your main language is the language you use most naturally. It could be the language you use at home.',
     },
   ],
 };
-
-local englandOptions = {
-  options: [
-    {
-      label: 'English',
-      value: 'English',
-    },
-    {
-      label: 'Other',
-      value: 'Other',
-      description: 'Including British Sign Language',
-      detail_answer: {
-        id: 'language-answer-other',
-        type: 'TextField',
-        mandatory: false,
-        label: 'Please specify main language',
-      },
-    },
+local proxyTitle = {
+  text: 'What is <em>{person_name_possessive}</em> main language?',
+  placeholders: [
+    placeholders.personNamePossessive,
   ],
 };
-
-local walesOptions = {
-  options: [
+local proxyDefinition = {
+  title: 'What do we mean by “main language”?',
```

Comment:
This is duplicated in the proxy and non-proxy version, it could be simplified to just having the content description as a variable.